### PR TITLE
dev: Spit out listening ports when running devservices

### PR DESCRIPTION
**Pull request by mattrobenolt**
image

### CODE REVIEW

File: src/sentry/runner/commands/devservices.py [hunk 1]
```diff
@@ -114,7 +114,10 @@ def up(project, exclude):
         else:
             container.stop()
             container.remove()
-        click.secho("> Creating '%s' container" % options['name'], err=True, fg='yellow')
+        listening = ''
+        if options['ports']:
+            listening = ' (listening: %s)' % ', '.join(map(str, options['ports'].values()))
```
Comment (tkaemming) [inline src/sentry/runner/commands/devservices.py]: Maybe sort these?

**Review by tkaemming — APPROVED** [review]
Can confirm, these are in fact ports

**Comment by tkaemming** [comment]
Also, if you just followed the lint failure in the test suite this would break hilariously when we upgrade to Python 3 so I guess just map over `'{}'.format` instead? lol

File: src/sentry/runner/commands/devservices.py (hunk 1 above)
Comment (mattrobenolt) [reply src/sentry/runner/commands/devservices.py]: Maybe.

**Pull request merged**